canonical/vanilla-monorepo
# Prune react app from storybook

Prunes the `App` react application from the base storybook

## QA
1. `bun install`
2. `cd ds/app`
3. `bun run storybook`
4. Open the storybook and see that the pages load and look as expected
